lewis/adapters/stream.py

```diff
@@ -86,7 +84,44 @@ def handle_accept(self):
             StreamHandler(sock, self.target)
 
 
-class Cmd(object):
+class CanNotProcessException(Exception):
+    pass
+
+
+class BaseCommand(object):
+    member = None
+    argument_mappings = None
+    return_mapping = None
```

Comment:
`_override`, `_transform`, `_mutate`, `_process`, `_filter` (I wouldn't say this necessarily removes; consider audio/video/image filters)